bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/values/connector/AsyncFunctionCallBack.java

```diff
@@ -69,7 +69,7 @@ public void notifySuccess() {
 
     @Override
     public void notifyFailure(ErrorValue error) {
-        RuntimeUtils.handleRuntimeErrors(error);
+        RuntimeUtils.handleRuntimeErrorsAndExit(error);
```

Comment:
is it correct to call exit here? then jvm would exit right?

Reply:
@HindujaB @warunalakshitha

Reply:
This seems bug. lets just put just "handleRuntimeErrors" without exit.

Reply:
Modified them